### PR TITLE
Add a prow pr-kops-aws job

### PR DESCRIPTION
**Pull request by krzyzacy**
One step closer to Jenkins death

(also fixed up kops jobs e2e-prow image)

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -400,6 +400,77 @@ presubmits:
...
+        args:
+        - --root=/go/src
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=k8s.io/release"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
```
Comment (BenTheElder) [inline prow/config.yaml]: tune the timeout maybe?

**Comment by BenTheElder** [comment]
one possible nit otherwise 
/lgtm
Soon we will be far from cleaning up Jenkins :smiling_imp:

File: prow/config.yaml (hunk 1 above)
Comment (krzyzacy) [reply prow/config.yaml]: matches Jenkins timeout now

**Comment by krzyzacy** [comment]
@ixdy one more mysterious bazel build - which somehow able to produce a dirty tag (twice), we might need to do `bazel clean` before `bazel build` :-(
```
I0922 01:12:23.693] push-build.sh: BEGIN main on 85531db9-9f31-11e7-b9ee-0a580a6c0209 Fri Sep 22 01:12:23 UTC 2017
I0922 01:12:23.694] 
I0922 01:12:24.235] Refusing to push dirty build with --ci flag given.
I0922 01:12:24.239] CI builds should always be performed from clean commits.
I0922 01:12:24.244] 
I0922 01:12:24.247] kubectl version output:
I0922 01:12:24.250] Client Version: version.Info{Major:"1", Minor:"9+",
I0922 01:12:24.251] GitVersion:"v1.9.0-alpha.0.862+240daf6e3596be-dirty",
I0922 01:12:24.251] GitCommit:"240daf6e3596bed5f55c4b67a6ee22986e076d9c", GitTreeState:"dirty",
```

**Comment by ixdy** [comment]
@krzyzacy where did you see that one?

**Comment by BenTheElder** [comment]
@ixdy we should probably move this to the issue above (?), but [here](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/46662/pull-kubernetes-e2e-kops-aws-prow/2/) is a kops on prow jobw with the dirty tree issue 😕

**Comment by BenTheElder** [comment]
Noting that the issue(s) were discovered by @ixdy and are discussed in #4693 for future reference.

**Pull request merged**